sys/include/net/netopt.h

```diff
@@ -645,6 +645,23 @@ typedef enum {
      */
     NETOPT_LORAWAN_MIN_RX_SYMBOL,
 
+    /**
+     * @brief   (uint8_t*) phy layer syncword
+     */
+    NETOPT_SYNCWORD,
+
+    /**
+     * @brief  (uint32_t) Get a random value from the device
+     *
+     * Nothing happens when set
+     */
+    NETOPT_RANDOM,
+
+    /**
+     * @brief (uint8_t) Get or set the number of PHY symbols before assuming there's no data
+     */
+    NETOPT_RX_SYMBOL_TIMEOUT,
```

Comment:
same as above, may need prefix `LORAWAN`?

Reply:
same as above, this is not LoRaWAN specific, even if only if LoRa requires it for the moment in RIOT.